gallery/src/components/NxButton/NxButtonPage.tsx

```diff
@@ -67,14 +68,16 @@ export default function NxButtonPage() {
                           id="nx-button-icon-example"
                           liveExample={NxButtonIconExample}
                           codeExamples={nxButtonIconCode}>
-        An example of a button containing an icon in addition to text.
+          An example of a button containing an icon in addition to text. For accessibility purposes it is important to
+          add an aria-label and aria-hidden=false for screen reader to interpret the content correctly.
```

Comment:
```suggestion
        An example of a button containing an icon in addition to text. For accessibility purposes it is important to
        add an aria-label and aria-hidden=false for screen reader to interpret the content correctly.
```

Reply:
5e921c6360b0259ddb2c519f27d920dbbcf1214d

Reply:
> 5e921c6

Indentation is wrong

Reply:
884ee01942d0d18b18de8dc651897a1e3ad29f02